### DISCUSSION TITLE
Is it reasonable to try to load billions of triples (20B+) to Corese?

### DISCUSSION THREAD

**Discussion by AndreaWesterinen** [Wimmics/corese#95] [Q&A]
I would like to experiment with the Wikidata RDF dump, but don't want to waste time if data this large (50G+) is not a target for Corese.

Thank you.

Andrea

**Comment by ocorby** [comment]
Hi, 
To my knowledge, 50G+ is too much. 
Best regards 
Olivier

**Reply by AndreaWesterinen** [reply]
Thanks.  I could try to split the data load, and maybe split the triples across several stores, but each store would need to hold 10-15B triples. 

Is that doable?
Andrea

**Comment by ocorby** [comment]
Hello, 

No, it will not work with B triples. 
It would work between 50M~100M triple depending on the memory. 

Best, 
Olivier
